src/System.IO.Pipelines/src/System/IO/Pipelines/ThreadPoolScheduler.NetStandard.cs

```diff
@@ -11,12 +11,12 @@ internal sealed class ThreadPoolScheduler : PipeScheduler
     {
         public override void Schedule(Action action)
         {
-            Task.Factory.StartNew(action);
+            Task.Factory.StartNew(action, CancellationToken.None, TaskCreationOptions.DenyChildAttach, TaskScheduler.Default);
```

Comment:
If the goal is to prefer the global queue, you can just use TP.QUWI here?

Reply:
I wasn’t sure if that was available on netstandard 1.1 (which I believe is what we support)

Reply:
Ah